draft-irtf-cfrg-rsa-blind-signatures.md

```diff
@@ -136,7 +148,8 @@ in {{!RFC8017}} to enable blind signature support.
 
 Originally introduced in the context of digital cash systems by Chaum
 for untraceable payments {{Chaum83}}, RSA blind signatures turned out to have
-a wide range of applications ranging from electronic voting schemes to authentication mechanisms.
+a wide range of applications ranging from privacy-preserving digital payments to
+authentication mechanisms {{GoogleVPN}} {{ApplePrivateRelay}} {{PrettyGoodPhonePrivacy}}.
```

Comment:
Private Relay uses this standard versus Google VPN that uses FDH I believe. But for this paragraph I don't think it makes a difference.

Reply:
Yeah, I figured the encoding was an irrelevant detail at this point.